autoload/gutentags.vim

```diff
@@ -242,6 +242,7 @@ function! gutentags#get_execute_cmd() abort
         if g:gutentags_background_update
             let l:cmd .= '/b '
         endif
+        let l:cmd .= '"'
```

Comment:
Hi! Do you need this change and the one in the "suffix" function? Isn't the `shellescape` change the only thing you need?